vividus-plugin-excel/src/main/java/org/vividus/bdd/steps/ExcelFileSteps.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2019-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.bdd.steps;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.IntStream;
+
+import javax.inject.Inject;
+
+import org.apache.poi.ss.usermodel.Row;
+import org.apache.poi.xssf.usermodel.XSSFSheet;
+import org.apache.poi.xssf.usermodel.XSSFWorkbook;
+import org.jbehave.core.annotations.When;
+import org.jbehave.core.model.ExamplesTable;
+import org.vividus.bdd.context.IBddVariableContext;
+import org.vividus.bdd.variable.VariableScope;
+
+public class ExcelFileSteps
+{
+    @Inject
+    private IBddVariableContext bddVariableContext;
+
+    /**
+     * Creates temporary xlsx file with specified content and puts path to that file to variable with specified name.
+     * Created file will be removed while termination of the JVM
+     * @param scopes The set (comma separated list of scopes e.g.: STORY, NEXT_BATCHES) of variable's scope<br>
+     * <i>Available scopes:</i>
+     * <ul>
+     * <li><b>STEP</b> - the variable will be available only within the step,
+     * <li><b>SCENARIO</b> - the variable will be available only within the scenario,
+     * <li><b>STORY</b> - the variable will be available within the whole story,
+     * <li><b>NEXT_BATCHES</b> - the variable will be available starting from next batch
+     * </ul>
+     * @param pathVariable name of variable
+     * @param name name of temporary file
+     * @param exampleTable any valid ExamplesTable that will be written to the file
+     * @throws IOException if an I/O exception of some sort has occurred
+     */
+
+    @When("I initialize the '$scopes' variable '$pathVariable' using path of temporary xlsx file"
+            + " with name '$name' and content: $examplesTable")
```

Comment:
questions:
- sheet name - is it not needed?
- should it really be temporary file?

Reply:
sheet name is not needed
file can be temporary